tests/xorg-test-run.sh

```diff
@@ -1,13 +1,25 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
Can we stick with `sh` please? Some systems, notable FreeBSD may not have bash installed.